### ISSUE TITLE
[BUG] Get phrase vector language switching problem

### ISSUE THREAD

**Issue by AlexKay28** [AlexKay28/langflow#118]
Current Behavior:
1 query to http://localhost:6769/get_phrase_vector - 200OK
2 query - langflow_application_nlp exited with code 247 

Traceback:
```
langflow_application_nlp | Warning : `load_model` does not return WordVectorModel or SupervisedModel any more, but a `FastText` object which is very similar.
langflow_application_nlp | 10.192.1.1 - - [31/May/2022 17:51:39] "GET /get_phrase_vector HTTP/1.1" 200 -
langflow_application_nlp exited with code 247
```

Expected Behavior:
200OK when I switch the language to vectorize

Steps To Reproduce:
Try to send request to vectorize any sentence for any language. Its going to be successfully, but after switch the language and it will be broken

Environment:
OS: macos
docker: 20.10.12
docker-compose: 1.29.2
python: 3.7.9

**Comment by AlexKay28** [comment]
The problem is going to be resolved in #117

**Comment by AlexKay28** [comment]
Observe uexpected behaviour. It might be pure connection problems.
Added retries to the facade API. It will help to avoid connection problems temporarily
